### DISCUSSION TITLE
Deleting every trace of skyline

### DISCUSSION THREAD

**Discussion by 8knestyl** [skyline-emu/skyline#1168] [General]
App was running fine until I watched a YouTube video and decided to update the app and title/production key. Now the app crashes on title screen of Sonic everytime. I tried to used the previous APK and now it crashes too. Is there a folder that has cookies or cache on skyline that a simple uninstall of the app won't rid?

**Comment by Crytonics** [comment]
What phone do you have?

**Comment by PixelyIon** [comment]
There is an OS level backup if you checked the checkbox to save all data while uninstalling, there are dedicated issues on how to uninstall Skyline in that case but the easiest solution is to use adb and run `adb uninstall skyline.emu` but if you can't do that then I think using `pm uninstall skyline.emu` in Termux works too.
